code/modules/admin/admin_verbs.dm

```diff
@@ -801,6 +802,41 @@ GLOBAL_PROTECT(admin_verbs_hideable)
 		can.valve_open = FALSE
 		can.update_icon()
 
+/client/proc/delete_book()
+	set category = "Admin"
+	set name = "Delete Book"
+
+	if(!check_rights(R_ADMIN))
+		return
+
+	var/bookid = input(usr, "What Book ID would you like to remove:", "Literally Fahrenheit 451") as null|num
+	if(!bookid)
+		return
+	
+	var/datum/DBQuery/query_library_print = SSdbcore.NewQuery(
```

Comment:
Check if db is connected and early return before this line

Reply:
That's built in to the datum. Although, it might be good to have specific error handling, which could be managed by an early return, for this since right now it'll just return that the book ID can't be found.